packages/amesos2/src/Amesos2_Superlu_TypeMap.hpp

```diff
@@ -79,294 +74,9 @@ extern "C" {
 
 #undef __SUPERLU_SUPERMATRIX
 #include "supermatrix.h"	// for Dtype_t declaration
-
-#ifdef HAVE_TEUCHOS_COMPLEX
```

Comment:
The TypeMap is to go from one type to another type. FunctionMap to map functions. Looks like this was moved to FunctionMap.

Reply:
Same questions as the 1st comment - if the changes I did in TypeMap are bad then I'll be reverting some stuff. But if the new view solvers will ever store data of native types I will need to discuss more how we will do that.